.eslintrc

```diff
@@ -94,3 +94,4 @@ rules:
   "no-path-concat": 0
   "no-process-exit": 0
   "eol-last": 2
+  "key-spacing": 0
```

Comment:
Well, running `grunt eslint` with `key-spacing: 1` in the .eslintrc file shows that we get about 72 warnings (all in test/*):
```
$ grunt eslint
Running "eslint:src" (eslint) task

test/functional_test.js (47 results)
test/storage_test.js (24 results)
test/tokbox_test.js (1 results)

✖ 72 problems (0 errors, 72 warnings)
```

It looks like because of [/test/storage_test.js:37-45](https://github.com/mozilla-services/loop-server/blob/b71f3731f36adc4e8c8af3facfa1c0eb4f38a2ea/test/storage_test.js#L37-L45) which aligns by value:
```js
        {
          callId:       randomBytes(16).toString("hex"),
          callerId:     callerId,
          userMac:      userMac,
          sessionId:    fakeCallInfo.session1,
          calleeToken:  fakeCallInfo.token1,
          callState:    constants.CALL_STATES.INIT,
          timestamp:    now - 3
        },
```

Or a couple places where we have two spaces after a colon:
```js
        {
          timestamp:  now,
          expires: now + callUrls.timeout,
          userMac: userMac
        },
```

If I change the eslint rule to `key-spacing: [1, {align: value}]` in the .eslintrc, it gets really sad and spits out:
`✖ 818 problems (0 errors, 818 warnings)`.

Since the `key-spacing` is only inconsistent in the 3 `test/{functional,storage,tokbox}_test.js` files, I'd probably vote to tweak those inconsistencies and then set the `key-spacing` to `1`.

I'm OK with turning off this rule now so we can merge this beautiful PR, and I'll submit a separate PR which sets `key-spacing: 1` and I'll tweak the inconsistent test/* files so they pass. Unless somebody objects, in which case I'll shut up.